iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -23,47 +30,75 @@ public enum ErrorCode
         BulkRegistryOperationFailure = 400013,
         CannotRegisterModuleToModule = 400301,
 
-        //Unauthorized - 401
+        /// <summary>
+        /// Unauthorized - 401
+        /// </summary>
         IotHubNotFound = 401001,
```

Comment:
Since the naming of this enum is questionable at best, I'd add to the comment that "This error code signals that the cloud to device operation failed because the device is either not registered, or is disabled"

https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-troubleshoot-error-404001-devicenotfound

Reply:
Cool I think a link to that will also be good to add.